src/main/java/com/example/Taitokysely/model/Survey.java

```diff
@@ -23,14 +23,15 @@
 	// PVM
 	private String surveyDate;
 
-	// EN TIIÄ TOIMIIKO VIELÄ, MUTTA IDEANA OLISI SAADA TÄHÄN MEIDÄN SURVEY LUOKKAAN
-	// TALTEEN KYSYMYKSEN JA VASTAUKSET LISTAMUODOSSA
-
-	// OLI ONGELMAA, ETTÄ TULI JOTAIN BEAN ONGELMIA MUTTA GOOGLAILUN JÄLKEEN TÄMÄ ELEMENT COLLECTION RATKAISI ONGELMAN
-	@ElementCollection
+	
+	//TÄHÄN JÄIN
```

Comment:
Jes, eli tämän Survey ja SurveyRepositoryn olisi voinut myös jättää tästä PR:stä poiskin ja pistää vaan toimivat osuudet tai sitten kommentoida nämä ei toimivat osuudet toistaseksi kokonaan pois, nyt näyttää siltä että näitä yritetään jo käyttää? Käytetäänkö näitä oikeasti vielä, jos ei ni vähintään kommentoituna pois etten turhaan ihmettele rikkinäistä koodia